src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -750,7 +759,7 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
      *
      * This function should be called when the room members or call memberships might have changed.
      */
-    public onMembershipUpdate = (): void => {
+    public onMembershipsUpdate = (): void => {
```

Comment:
This currently makes up part of the public/exported API. So, rather than just renaming I think it should be left as is or deprecate the existing one and add an override.